### ISSUE TITLE
Build opencv with cuda support

### ISSUE THREAD

**Issue by innerlee** [JuliaPackaging/Yggdrasil#331]
Currently cuda cannot be found by opencv. Perhaps we need `nvcc` etc. Can we remove this line https://github.com/JuliaPackaging/Yggdrasil/blob/master/C/CUDA/build_tarballs.jl#L43 ?

**Comment by giordano** [comment]
I don't think we currently really support building libraries with support for CUDA.

**Comment by innerlee** [comment]
:(

**Comment by ViralBShah** [comment]
Would be nice to support building stuff with CUDA support.

**Comment by ViralBShah** [comment]
This should now be straightforward to do for someone to give it a shot. There's lots of packages with CUDA support.
